app/src/main/java/org/wikipedia/talk/TalkTopicActivity.kt

```diff
@@ -295,6 +305,7 @@ class TalkTopicActivity : BaseActivity(), LinkPreviewDialog.Callback {
     private fun onSaveClicked() {
         val subject = binding.replySubjectText.text.toString().trim()
         var body = binding.replyEditText.text.toString().trim()
+        undoneText = binding.replyEditText.text.toString().trim()
```

Comment:
Looks like it can be `undoneText = body` instead of getting the text from `replyEditText` again.